### PR TITLE
Milestone: Navigation

### PR DESCRIPTION
**Pull request by newhorizon-tech**
- Single Page Application 
- Navigation Bar 
- Dynamically generated content

### CODE REVIEW

**Review by smcommits — COMMENTED** [review]
Hi @newhorizon-tech, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please make sure that all your linter checks are passing. Also, please make sure to configure all the required linter for the project. Your repo is missing the `Lighthouse` and `Webhint` linter. For any project that uses HTML, CSS, and Javascript, you have to follow [these instructions](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to set up the linters. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by newhorizon-tech** [comment]
Changes made 

- Fixed ESlint errors
- Added Lighthouse and Webhint Linters

File: index.js [hunk 1]
```diff
@@ -103,3 +110,35 @@ const bookInput = () => {
 };
 
 addBtn.addEventListener('click', bookInput);
```
Comment (sinansevgi) [inline index.js]: [Optional]
It will be better if you change the page to the book list after adding a new book.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @newhorizon-tech , 

Linters configured correctly and all checks are passing 👍 
Navigation works well👍 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (newhorizon-tech) [reply index.js]: Done

**Pull request merged**